Gemfile.lock

```diff
@@ -28,6 +28,14 @@ GIT
       nokogiri (~> 1)
       rspec (>= 3.0.0.a, < 4)
 
+GIT
+  remote: https://github.com/3scale/sidekiq-batch.git
+  revision: f1c56d0b8445fd3eb337614aedba1752aa19938e
+  branch: redis-client
```

Comment:
Does this have a native component? It will requre some sort of a fix for prod images.

Reply:
No native component, and they have progressed on supporting Sidekiq 7 but still haven't launched a release. I hope they do it soon so we can get rid of the fork.